osbenchmark/benchmark.py

```diff
@@ -221,6 +222,14 @@ def add_workload_source(subparser):
         help="Whether to include the comparison in the results file.",
         default=True)
 
+    aggregate_parser = subparsers.add_parser("aggregate", help="Aggregate multiple test_executions")
+    aggregate_parser.add_argument(
+        "--test-executions",
+        "--t",
+        type=non_empty_list,
+        required=True,
+        help="Comma-separated list of TestExecution IDs to aggregate")
+
```

Comment:
We should include other common options like `--test-execution-id` and `--results-file`

Reply:
Added these! Let me know if I should add others as well